README.md

```diff
@@ -7,7 +7,11 @@
 Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
 ```text
-Your answer...
+SQL databases are relational while NoSQL databases aren't. SQL databases use tables while NoSQL databases use documents, key-value pairs and graphs (This means it doesn't have to adhere to a schema).
+
+SQL databases are vertically scalable while NoSql is horizontal.
+
+If you have a lot of data its better to use NoSQL, because the query time for SQL is really long
```

Comment:
:+1:  Good description.  The last statement isn'y necessarily true.  Relational database are highly optimized, but it can take a  little longer if you have a lot of joins.